lang/src/test/kotlin/org/partiql/lang/syntax/PartiQLParserMetaTests.kt

```diff
@@ -51,6 +51,6 @@ internal class PartiQLParserMetaTests : PartiQLParserTestBase() {
         val inCollection = ast.expr as PartiqlAst.Expr.InCollection
         val list = inCollection.operands[1] as PartiqlAst.Expr.List
 
-        assert(list.metas.containsKey(IsListParenthesizedMeta.TAG))
```

Comment:
metas.containsKey is kind of funny. You'd think there would be a metas.contains(meta) which sure might be backed by the string lookup. Why define an interface if you aren't going to use it 🤷